src/commands/edit-profile/index.ts

```diff
@@ -54,20 +51,17 @@ export class EditProfileCommand extends Command {
           );
         }
 
-        await userFirestoreRef.set({ about }, { merge: true });
+        await setDBGuildMember({ about });
```

Comment:
TODO update this to collect all fields to update and only make a single `setDBGuildMember` call